internal/auth_request/model/request.go

```diff
@@ -19,7 +19,8 @@ const (
 )
 
 const (
-	OrgDomainPrimaryScope = "urn:zitadel:org:domain:primary:"
+	OrgDomainPrimaryScope = "urn:zitadel:iam:org:domain:primary:"
+	ProjectIDAudScope     = "urn:zitadel:iam:org:project:id:aud:"
```

Comment:
@fforootd didn't we state to go with: `urn:zitadel:iam:org:project:{projectid}:aud`